File: spring-boot-java-web/src/main/java/com/example/webjsp/model/City.java
package com.example.webjsp.model;

import com.fasterxml.jackson.databind.ObjectMapper;
import lombok.Getter;
import lombok.Setter;

import java.sql.Timestamp;
import java.util.UUID;

@Getter
@Setter
public class City {
  UUID id;
  String name;
  String state;
  String country;
  Timestamp insTime;

  @Override
  public String toString() {
    try {
      ObjectMapper mapper = new ObjectMapper();
      return mapper.writeValueAsString(this);
    } catch (Exception ex) {
      return "";
    }
  }
}

File: spring-boot-kotlin-web/src/main/kotlin/com/example/webjsp/model/City.kt
package com.example.webjsp.model

import java.sql.Timestamp
import java.util.UUID

/**
 * Data VO of City
 *
 * City 정보를 담는 data 클래스
 *
 */
data class City(
  // ID
  var id: UUID,
  // 시/군/구 명
  var name: String,
  // 시/도 명
  var state: String,
  // 국가명
  var country: String,
  // 등록일시
  var insTime: Timestamp
)
